dusty/commands/repos.py

```diff
@@ -69,9 +81,12 @@ def update_managed_repos(force=False):
     specs_repo = get_specs_repo()
     if not specs_repo.is_overridden:
         log_to_client('Updating managed copy of specs-repo before loading specs')
+        add_known_hosts_for_repos([specs_repo])
```

Comment:
still not the right place to do it.  Could do it in up before we call update_managed_repos?